.ci/build

```diff
@@ -22,46 +31,63 @@ fi
+        --from-image ubuntu:18.04 \
+        --title "gardener shell" \
+        --dockerfile "${GENERATED_DOCKERFILES_PATH}"/ops-toolbelt-az-k$i.dockerfile
 
-"${SOURCE_PATH}"/generator/generate-dockerfile.py \
-    --dockerfile-configs "${SOURCE_PATH}"/dockerfile-configs/common-components.yaml \
-    --dockerfile-configs "${SOURCE_PATH}"/dockerfile-configs/openstack-components.yaml \
-    --dockerfile-configs "${SOURCE_PATH}"/dockerfile-configs/gardenctl-components.yaml \
-    --from-image ubuntu:18.04 \
-    --title "gardener shell" \
-    --dockerfile "${GENERATED_DOCKERFILES_PATH}"/ops-toolbelt-gardenctl-openstack.dockerfile
+    "${SOURCE_PATH}"/generator/generate-dockerfile.py \
+        --dockerfile-configs "${SOURCE_PATH}"/dockerfile-configs/common-components.yaml \
+        --dockerfile-configs "${SOURCE_PATH}"/dockerfile-configs/gcp-components.yaml \
+        --dockerfile-configs "${SOURCE_PATH}"/dockerfile-configs/gardenctl-components.yaml \
+        --dockerfile-configs "${SOURCE_PATH}"/dockerfile-configs/kubectl-components.yaml \
+        --from-image ubuntu:18.04 \
+        --title "gardener shell" \
+        --dockerfile "${GENERATED_DOCKERFILES_PATH}"/ops-toolbelt-gcp-k$i.dockerfile
 
-"${SOURCE_PATH}"/generator/generate-dockerfile.py \
-    --dockerfile-configs "${SOURCE_PATH}"/dockerfile-configs/common-components.yaml \
-    --dockerfile-configs "${SOURCE_PATH}"/dockerfile-configs/aliyun-components.yaml \
-    --dockerfile-configs "${SOURCE_PATH}"/dockerfile-configs/gardenctl-components.yaml \
-    --from-image ubuntu:18.04 \
-    --title "gardener shell" \
-    --dockerfile "${GENERATED_DOCKERFILES_PATH}"/ops-toolbelt-gardenctl-aliyun.dockerfile
\ No newline at end of file
+    "${SOURCE_PATH}"/generator/generate-dockerfile.py \
+        --dockerfile-configs "${SOURCE_PATH}"/dockerfile-configs/common-components.yaml \
+        --dockerfile-configs "${SOURCE_PATH}"/dockerfile-configs/openstack-components.yaml \
+        --dockerfile-configs "${SOURCE_PATH}"/dockerfile-configs/gardenctl-components.yaml \
+        --dockerfile-configs "${SOURCE_PATH}"/dockerfile-configs/kubectl-components.yaml \
+        --from-image ubuntu:18.04 \
+        --title "gardener shell" \
+        --dockerfile "${GENERATED_DOCKERFILES_PATH}"/ops-toolbelt-openstack-k$i.dockerfile
+
+    "${SOURCE_PATH}"/generator/generate-dockerfile.py \
+        --dockerfile-configs "${SOURCE_PATH}"/dockerfile-configs/common-components.yaml \
+        --dockerfile-configs "${SOURCE_PATH}"/dockerfile-configs/aliyun-components.yaml \
+        --dockerfile-configs "${SOURCE_PATH}"/dockerfile-configs/gardenctl-components.yaml \
+        --dockerfile-configs "${SOURCE_PATH}"/dockerfile-configs/kubectl-components.yaml \
+        --from-image ubuntu:18.04 \
+        --title "gardener shell" \
+        --dockerfile "${GENERATED_DOCKERFILES_PATH}"/ops-toolbelt-aliyun-k$i.dockerfile
+done
```

Comment:
I assume the `kubectl-components.yaml` file is only used temporarily to generate the dockerfile, right? Shouldn't this file be cleaned up afterwards? It anyhow only contains the contents of the last run of the loop